lib/ssf/client.rb

```diff
@@ -27,69 +27,25 @@ def send_to_socket(message)
       @socket.send(message, 0)
     end
 
-    def new_root_span(operation: '', tags: {})
+    def start_span(operation: '', tags: {}, trace_id: nil, parent_id: nil)
```

Comment:
I'd still strongly recommend changing this to take in an SSF as the parent rather the trace_id and parent_id. In most cases, that'll be much more convenient - the one exception is at a service boundary, where we're extracing the ID values from HTTP headers, but in that case, we could add a `reconstruct_span` helper function, either here or in our internal wrapper to automatically extract + start a new span from the headers.